### PR TITLE
reference: update statement summary tables

### PR DESCRIPTION
**Pull request by djshow832**
### What is changed, added or deleted? (Required)

- Reorder the content
- Add description for cluster tables
- Update table names and fields

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by yikeke** [comment]
This branch will be cherry-picked to release-4.0 branch after the Doc Special Week, so no need to add the needs-cherry-pick-4.0 label now. FYI. @djshow832

File: statement-summary-tables.md [hunk 1]
```diff
@@ -74,7 +79,109 @@ select * from employee where id in (...) and salary between ? and ?;
...
+- 优先读 session 变量，没有设置过 session 变量才会读 global 变量
+- 把 session 变量设为空字符串，将会重新读 global 变量
+
+> **注意：**
+>
+> `tidb_stmt_summary_history_size`、`tidb_stmt_summary_max_stmt_count`、`tidb_stmt_summary_max_sql_length` 这些配置都影响内存占用，建议根据实际情况调整，不宜设置得过大。
+
+## 目前的限制
+
+Statement summary tables 现在还存在一些限制：
```
Comment (TomShawn) [inline statement-summary-tables.md:128]:
```suggestion
Statement summary tables 现在还存在以下限制：
```

**Comment by TomShawn** [comment]
@bb7133 PTAL

**Pull request merged**